SEED-platform/seed
# Redirect on org change if necessary

#### Any background context you want to provide?
When changing the organization from the menu, the page did not redirect to that org's data if the url contained an `organization_id` argument.

#### What's this PR do?
checks for the presence of the `organization_id` arg and updates it if necessary

#### How should this be manually tested?
With multiple orgs navigate to any of the organization settings pages. Change the org from the dropdown. The page data should reflect the newly selected org.

#### What are the relevant tickets?
#4575

#### Screenshots (if appropriate)
![Screenshot 2024-03-19 at 1 17 45 PM](https://github.com/SEED-platform/seed/assets/58446472/a399ce09-caab-491d-bf32-e251a3c2b2f7)
